app/src/main/res/values/strings.xml

```diff
@@ -138,4 +138,12 @@
     <string name="end_hour">End hour:</string>
     <string name="price">Price:</string>
     <string name="update_success">Update Success</string>
+    <string name="All">All</string>
+    <string name="mi_quang">Mì Quảng</string>
+    <string name="com">Cơm</string>
+    <string name="nep_cam">Nếp Cẩm</string>
+    <string name="top_shop">Top Shop</string>
+    <string name="categories">Categories</string>
+    <string name="view_all">View All</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.view_all` appears to be unused
 Unused resources make applications larger and slow down builds.